README.md

```diff
@@ -140,10 +128,10 @@ test/
      test-geoip-netspeed.c
 ```
 
-The test-geoip.c program works with both the GeoLite Legacy and GeoIP Legacy
-Country databases. The test-geoip-city.c program works with both the GeoLite
-Legacy and GeoIP Legacy City databases. The other example programs require the
-paid databases available (https://www.maxmind.com/en/geolocation_landing).
+The test-geoip.c program works with the GeoIP Legacy Country database. The
+test-geoip-city.c program works with the GeoIP Legacy City databases. The
+other example programs require the paid databases available
```

Comment:
"The other example programs require the paid databases available (<url>)." seems like a weird sentence to me. What do you think?

Reply:
Yeah, I'd say 50% of this file is compromised of weird sentences. I'll rewrite this one though.